First pull request

ilijalazarevic opened the pull request.

Commit:
Initial version without styles
	new file:   controllers/maincontroller.js
	new file:   directives/checkpresence.js
	new file:   services/ExceptionHandler.js
	modified:   config/routes.js
	modified:   controllers/logincontroller.js
	modified:   filters/trans.js
	modified:   public/app.js
	modified:   services/Login.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
Modified existing files
	modified:   config/routes.js
	modified:   controllers/logincontroller.js
	modified:   filters/trans.js
	modified:   public/app.js
	modified:   services/Login.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
Fixed problem with getFeedItems function
	modified:   controllers/maincontroller.js
	modified:   public/app.js
	modified:   services/Login.js
	modified:   templates/main.html

Commit:
Introducing design and slight changes in html structures.
	modified:   controllers/maincontroller.js
	modified:   directives/checkpresence.js
	new file:   filters/titleshortener.js
	modified:   filters/trans.js
	modified:   public/app.js
	modified:   services/Login.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
modified:   Gruntfile.js
	modified:   controllers/logincontroller.js
	modified:   controllers/maincontroller.js
	modified:   directives/checkpresence.js
	modified:   filters/titleshortener.js
	modified:   filters/trans.js
	modified:   public/app.js
	modified:   services/Login.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html
	modified:   vendor/angular/angular-mocks.js
	modified:   vendor/angular/angular.js
	css/bootstrap.css
	css/mystyle.css
	css/responsive.css
	images/
	img/glyphicons-halflings-white.png
	img/glyphicons-halflings.png
	less/
	services/CorsService.js
	vendor/bootstrap/

Commit:
new file:   css/bootstrap.css
	new file:   css/mystyle.css
	new file:   css/responsive.css
	new file:   images/owncloud-logo-transparend-white.png
	new file:   images/owncloud-logo-transparend.png
	new file:   img/glyphicons-halflings-white.png
	new file:   img/glyphicons-halflings.png
	new file:   less/accordion.less
	new file:   less/alerts.less
	new file:   less/bootstrap.less
	new file:   less/breadcrumbs.less
	new file:   less/button-groups.less
	new file:   less/buttons.less
	new file:   less/carousel.less
	new file:   less/close.less
	new file:   less/code.less
	new file:   less/component-animations.less
	new file:   less/dropdowns.less
	new file:   less/forms.less
	new file:   less/grid.less
	new file:   less/hero-unit.less
	new file:   less/labels-badges.less
	new file:   less/layouts.less
	new file:   less/media.less
	new file:   less/mixins.less
	new file:   less/modals.less
	new file:   less/mystyle.less
	new file:   less/navbar.less
	new file:   less/navs.less
	new file:   less/pager.less
	new file:   less/pagination.less
	new file:   less/popovers.less
	new file:   less/progress-bars.less
	new file:   less/reset.less
	new file:   less/responsive-1200px-min.less
	new file:   less/responsive-767px-max.less
	new file:   less/responsive-768px-979px.less
	new file:   less/responsive-navbar.less
	new file:   less/responsive-utilities.less
	new file:   less/responsive.less
	new file:   less/scaffolding.less
	new file:   less/sprites.less
	new file:   less/tables.less
	new file:   less/tests/buttons.html
	new file:   less/tests/css-tests.css
	new file:   less/tests/css-tests.html
	new file:   less/tests/forms-responsive.html
	new file:   less/tests/forms.html
	new file:   less/tests/navbar-fixed-top.html
	new file:   less/tests/navbar-static-top.html
	new file:   less/tests/navbar.html
	new file:   less/thumbnails.less
	new file:   less/tooltip.less
	new file:   less/type.less
	new file:   less/utilities.less
	new file:   less/variables.less
	new file:   less/wells.less
	new file:   services/CorsService.js
	new file:   vendor/bootstrap/.jshintrc
	new file:   vendor/bootstrap/bootstrap-affix.js
	new file:   vendor/bootstrap/bootstrap-alert.js
	new file:   vendor/bootstrap/bootstrap-button.js
	new file:   vendor/bootstrap/bootstrap-carousel.js
	new file:   vendor/bootstrap/bootstrap-collapse.js
	new file:   vendor/bootstrap/bootstrap-dropdown.js
	new file:   vendor/bootstrap/bootstrap-modal.js
	new file:   vendor/bootstrap/bootstrap-popover.js
	new file:   vendor/bootstrap/bootstrap-scrollspy.js
	new file:   vendor/bootstrap/bootstrap-tab.js
	new file:   vendor/bootstrap/bootstrap-tooltip.js
	new file:   vendor/bootstrap/bootstrap-transition.js
	new file:   vendor/bootstrap/bootstrap-typeahead.js
	new file:   vendor/bootstrap/mylib.js
	new file:   vendor/bootstrap/tests/index.html
	new file:   vendor/bootstrap/tests/phantom.js
	new file:   vendor/bootstrap/tests/server.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-affix.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-alert.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-button.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-carousel.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-collapse.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-dropdown.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-modal.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-phantom.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-popover.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-scrollspy.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-tab.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-tooltip.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-transition.js
	new file:   vendor/bootstrap/tests/unit/bootstrap-typeahead.js
	new file:   vendor/bootstrap/tests/vendor/jquery.js
	new file:   vendor/bootstrap/tests/vendor/qunit.css
	new file:   vendor/bootstrap/tests/vendor/qunit.js

Commit:
Changes on hostname url
	modified:   public/app.js
	modified:   services/Login.js

Commit:
Renaming angular js scripts and deleting unnecessary ones
	renamed:    services/CorsService.js -> config/cors.js
	renamed:    controllers/logincontroller.js -> controllers/LoginController.js
	renamed:    controllers/maincontroller.js -> controllers/MainController.js
	renamed:    directives/checkpresence.js -> directives/CheckPresence.js
	renamed:    filters/trans.js -> filters/Translator.js
	deleted:    filters/titleshortener.js

Commit:
Changes in code concerning the coding standards and refactoring
	modified:   Gruntfile.js
	modified:   config/app.js
	modified:   config/routes.js
	modified:   config/testacular.js
	modified:   controllers/LoginController.js
	modified:   controllers/MainController.js
	modified:   directives/CheckPresence.js
	modified:   filters/Translator.js
	new file:   images/owncloud-logo-transparent-white.png
	new file:   images/owncloud-logo-transparent.png
	modified:   public/app.js
	modified:   services/ExceptionHandler.js
	new file:   services/FeedsService.js
	new file:   services/FoldersService.js
	new file:   services/ItemsService.js
	modified:   services/Login.js
	new file:   services/LoginService.js
	new file:   services/UserService.js
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
deleted:    images/owncloud-logo-transparend-white.png
	deleted:    images/owncloud-logo-transparend.png

Commit:
Basic login form field format control implemented
	modified:   controllers/LoginController.js
	modified:   controllers/MainController.js
	modified:   css/bootstrap.css
	modified:   css/mystyle.css
	modified:   less/mystyle.less
	modified:   public/app.js
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
Changed the way articles are listed, the way list is filled with new group of
new articles, and not refreshed with new ones.
Also, exceptions handling is improven.
	modified:   Gruntfile.js
	renamed:    services/ExceptionHandler.js -> config/exceptions.js
	modified:   controllers/LoginController.js
	modified:   controllers/MainController.js
	modified:   directives/CheckPresence.js
	modified:   public/app.js
	new file:   services/ExceptionsService.js
	modified:   services/FeedsService.js
	modified:   services/FoldersService.js
	modified:   services/ItemsService.js
	modified:   services/UserService.js
	modified:   templates/main.html

Commit:
Improved impelementation of listings heading captions
which is now opened for translations through filters
	modified:   controllers/MainController.js
	modified:   public/app.js
	modified:   templates/main.html

Commit:
Added option for switching languages, changed the translation filter, added Translation service etc.
	modified:   config/exceptions.js
	modified:   config/routes.js
	modified:   controllers/LoginController.js
	modified:   css/bootstrap.css
	modified:   css/mystyle.css
	modified:   directives/CheckPresence.js
	modified:   filters/Translator.js
	new file:   languages/en-us.json
	new file:   languages/sr-rs.json
	modified:   less/mystyle.less
	modified:   manifest.webapp
	modified:   public/app.js
	modified:   services/ExceptionsService.js
	modified:   services/LoginService.js
	new file:   services/TranslationService.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
Added new less design template and tweaked a bit look of the application
	modified:   controllers/MainController.js
	modified:   css/bootstrap.css
	modified:   css/mystyle.css
	new file:   css/mystyle1.css
	new file:   filters/UtilityFilters.js
	new file:   images/logo-white-transparent.png
	modified:   less/bootstrap.less
	modified:   less/mystyle.less
	new file:   less/mystyle1.less
	modified:   public/app.js
	modified:   services/FeedsService.js
	modified:   services/FoldersService.js
	modified:   services/ItemsService.js
	new file:   services/TimeService.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
Making some changes in services scripts for the sake of consistency.
Login.js is deprecated and is deleted.
	modified:   controllers/MainController.js
	modified:   public/app.js
	modified:   services/FeedsService.js
	modified:   services/FoldersService.js
	deleted:    services/Login.js
	modified:   services/LoginService.js
	modified:   services/TimeService.js
	modified:   templates/main.html

Commit:
Correcting way feeds listing shows date feed added.
	modified:   public/app.js
	modified:   services/FeedsService.js

Commit:
Adding mark read, mark favorite, delete functionalities for items.
	modified:   controllers/LoginController.js
	modified:   controllers/MainController.js
	modified:   css/bootstrap.css
	modified:   css/mystyle1.css
	modified:   less/mystyle1.less
	modified:   public/app.js
	modified:   services/FeedsService.js
	modified:   services/FoldersService.js
	modified:   services/ItemsService.js
	modified:   services/TimeService.js
	modified:   services/UserService.js
	modified:   templates/main.html

Commit:
Added directives for displaying lists with items, folders and feeds.
	modified:   controllers/MainController.js
	modified:   css/bootstrap.css
	modified:   css/mystyle1.css
	new file:   directives/FeedsListing.js
	new file:   directives/FoldersListing.js
	new file:   directives/ItemsListing.js
	modified:   less/mystyle1.less
	modified:   public/app.js
	modified:   services/ItemsService.js
	modified:   templates/index.html
	modified:   templates/main.html

Commit:
ItemsListing directive is now using ItemsService method for quering api
	modified:   css/bootstrap.css
	modified:   css/mystyle1.css
	modified:   directives/ItemsListing.js
	modified:   languages/en-us.json
	modified:   languages/sr-rs.json
	modified:   less/mystyle1.less
	modified:   public/app.js
	modified:   services/ExceptionsService.js
	modified:   templates/main.html

Commit:
Added directive for animated scrolling to the part of the same page.
	modified:   Gruntfile.js
	new file:   config/location.js
	modified:   controllers/MainController.js
	modified:   directives/ItemsListing.js
	new file:   directives/ScrollTo.js
	modified:   manifest.webapp
	modified:   public/app.js
	modified:   templates/main.html

Commit:
Slight changes is less files and various controller/filter scripts to adjust how page data is rendered.
	modified:   controllers/MainController.js
	modified:   css/bootstrap.css
	modified:   css/mystyle1.css
	modified:   css/responsive.css
	modified:   directives/FeedsListing.js
	modified:   directives/ScrollTo.js
	modified:   filters/UtilityFilters.js
	modified:   less/mystyle1.less
	modified:   less/responsive-767px-max.less
	modified:   less/responsive.less
	modified:   public/app.js
	modified:   templates/index.html
	modified:   templates/login.html
	modified:   templates/main.html

Commit:
Changes in app design
	modified:   css/bootstrap.css
	modified:   css/mystyle1.css
	modified:   css/responsive.css
	modified:   less/mystyle1.less
	modified:   less/responsive-768px-979px.less
	modified:   public/app.js
	modified:   services/TimeService.js
	modified:   templates/index.html
	modified:   templates/main.html

Commit:
Modified manifest file
	modified:   manifest.webapp

Commit:
Removed css files which are not needed
	deleted:    css/mystyle.css
	deleted:    css/mystyle1.css
	deleted:    css/style.css

Commit:
Changes in manifest file
	modified:   manifest.webapp

Commit:
Sorted index.html for better responsivenes concerning design.
	modified:   manifest.webapp
	modified:   templates/index.html

BernhardPosselt commented on www/config/location.js:
```diff
@@ -1,26 +1,25 @@
-}]);
\ No newline at end of file
+// define your routes in here
+angular.module('News').config(['$locationProvider', function ($locationProvider) {
```
This file can be removed it seems

ilijalazarevic replied:
Sure, it has no purpose right now.

BernhardPosselt commented:
Otherwise looks great, can you contact frank@owncloud.com for push access? Code is on a very high level, i like the frequent use of promises and services. Well done.

cc @frank

Commit:
Some small changes through files
	modified:   ../AUTHORS
	modified:   Gruntfile.js
	modified:   config/exceptions.js
	deleted:    config/location.js
	modified:   directives/ItemsListing.js
	modified:   public/app.js
	modified:   services/LoginService.js
	modified:   services/UserService.js
	modified:   templates/login.html
